webapp/lib/view.dart

```diff
@@ -208,41 +234,67 @@ class ContentView {
     singleIndicators.append(needsReplyAndEscalateMoreThan24hLatestValue.chartContainer);
 
     needsReplyTimeseries = new charts.DailyTimeseriesLineChartView();
-    contentElement.append(needsReplyTimeseries.chartContainer);
+    conversationChartsTabContent.append(needsReplyTimeseries.chartContainer);
     needsReplyTimeseries.createEmptyChart(
       titleText: 'needs reply',
       datasetLabels: ['needs reply']);
 
     needsReplyAndEscalateTimeseries = new charts.DailyTimeseriesLineChartView();
-    contentElement.append(needsReplyAndEscalateTimeseries.chartContainer);
+    conversationChartsTabContent.append(needsReplyAndEscalateTimeseries.chartContainer);
     needsReplyAndEscalateTimeseries.createEmptyChart(
       titleText: 'needs reply and escalate',
       datasetLabels: ['needs reply and escalate']);
 
     needsReplyMoreThan24hTimeseries = new charts.DailyTimeseriesLineChartView();
-    contentElement.append(needsReplyMoreThan24hTimeseries.chartContainer);
+    conversationChartsTabContent.append(needsReplyMoreThan24hTimeseries.chartContainer);
     needsReplyMoreThan24hTimeseries.createEmptyChart(
       titleText: 'needs reply more than 24h',
       datasetLabels: ['needs reply more than 24h']);
 
     needsReplyAndEscalateMoreThan24hTimeseries = new charts.DailyTimeseriesLineChartView();
-    contentElement.append(needsReplyAndEscalateMoreThan24hTimeseries.chartContainer);
+    conversationChartsTabContent.append(needsReplyAndEscalateMoreThan24hTimeseries.chartContainer);
     needsReplyAndEscalateMoreThan24hTimeseries.createEmptyChart(
       titleText: 'needs reply and escalate more than 24h',
       datasetLabels: ['needs reply and escalate more than 24h']);
 
     needsReplyAgeHistogram = new charts.HistogramChartView();
-    contentElement.append(needsReplyAgeHistogram.chartContainer);
+    conversationChartsTabContent.append(needsReplyAgeHistogram.chartContainer);
     needsReplyAgeHistogram.createEmptyChart(
       titleText: 'needs reply messages by date',
       datasetLabel: 'needs reply messages by date');
 
+    systemChartsTabContent = new DivElement()
+      ..id = "systems";
+
     restartSystemEventTimeseries = new charts.DailyTimeseriesLineChartView();
-    contentElement.append(restartSystemEventTimeseries.chartContainer);
+    systemChartsTabContent.append(restartSystemEventTimeseries.chartContainer);
     restartSystemEventTimeseries.createEmptyChart(
       titleText: 'system events',
       datasetLabels: ['restart']);
   }
+
+  void toogleTabView(ChartType chartType) {
+    contentElement.children.clear();
+    _systemTabLink.classes.remove('active');
+    _conversationTabLink.classes.remove('active');
+    switch(chartType) {
+      case ChartType.system:
+        contentElement.append(systemChartsTabContent);
+        _systemTabLink
+          ..classes.add('active');
```

Comment:
nit: no need for double-dot class access here and below

Reply:
ok...fixed. I've also made it so that the tabs are only visible when the user is signed in